### ISSUE TITLE
JSON Example Has No Match

### ISSUE THREAD

**Issue by colleend** [returntocorp/semgrep#2770]
**Describe the bug**
After upgrading to semgrep 0.43.0 (pushing the latest develop commits), the example on semgrep-app for JSON does not match with the target code.

**To Reproduce**
Try going to https://semgrep.dev/editor?version=0.43.0 -> load "Configuration Bugs (JSON)" and click Run.

**Expected behavior**
Expecting the match to happen.

**What is the priority of the bug to you?**
P1

**Comment by aryx** [comment]
Yes, it's because we now require some extra "..." in JSON to be more consistent with what we do in other languages.
With the extra ellipsis it works: https://semgrep.dev/s/3xgr/?version=0.43.0
Could you update the examples (no idea where the data is stored, probably somewhere in semgrep-app).

**Comment by aryx** [comment]
This is since https://github.com/returntocorp/semgrep/pull/2741
cc @mschwager
